how-to-guides/data-versioning/notebooks/Organize_and_share_dataset_versions.ipynb

```diff
@@ -41,9 +41,7 @@
     "* Share all the currently used dataset versions with your team\n",
     "* Assert that you are training on the latest dataset version available\n",
     "\n",
-    "By the end of this guide, you will log various dataset versions, organize them in the Neptune app and see how to share them with a persistent link.\n",
-    "\n",
-    "![image](https://neptune.ai/wp-content/uploads/neptune-datasets.png)"
+    "By the end of this guide, you will log various dataset versions, organize them in the Neptune app and see how to share them with a persistent link."
```

Comment:
All lines which are not the last line of a cell in notebooks need to end with a linebreak `\n`. Otherwise, this corrupts the file.

When making any changes to notebooks, it is best to do that in a rendered view itself and not the source view

```suggestion
    "By the end of this guide, you will log various dataset versions, organize them in the Neptune app and see how to share them with a persistent link.\n"
```

Reply:
Thank you. 

Adding Sourcery to my GitHub wants access to specific repo's, none of which are for Neptune, so I am not 100% sure how it works. 
![image](https://github.com/user-attachments/assets/ab1a2d8a-d345-482a-b9df-faa6f8cef0b9)

Reply:
> Adding Sourcery to my GitHub wants access to specific repo's, none of which are for Neptune, so I am not 100% sure how it works.

That's fine. All the suggestions happen in your local IDE. The GitHub permissions are only needed to verify your identity and check if the current project is open-source